components/kyverno/production/kflux-ocp-p01/kyverno-helm-values.yaml

```diff
@@ -24,8 +24,12 @@ admissionController:
 backgroundController:
   replicas: 3
   resources:
+    requests:
+      cpu: 200m
+      memory: 256Mi
```

Comment:
Changes look good but we should aim at request=limits, otherwise pod will get evicted if pod uses more than 256 and the node is packed